Sorts/CountingSort.js

```diff
@@ -8,31 +8,38 @@
  * Animated Visual: https://www.cs.usfca.edu/~galles/visualization/CountingSort.html
  */
 
-function countingSort (arr, min, max) {
-  let i
-  let z = 0
+const countingSort = (arr, min, max) => {
+  // Create an auxiliary resultant array
+  const res = []
+  // Create the freq array
   const count = []
+  let len = max - min + 1
 
-  for (i = min; i <= max; i++) {
+  for (let i = 0; i < len; i++) {
```

Comment:
Instead of doing the declaration and then the filling in the variable `count`, you can do:
```js
const count = (new Array(len)).fill(0)
```
If you do not understand how these work, refer to the following links:
1. [`Array()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Array)
2. [`Array.prototype.fill()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill)

Reply:
Hi @raklaptudirm, Thanks for reviewing. This is exactly what I wanted.